cinema/serializers.py

```diff
@@ -1 +1,99 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    CinemaHall,
+    Genre,
+    Actor,
+    Movie,
+    MovieSession,
+)
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ("id", "name", "rows", "seats_in_row", "capacity")
```

Comment:
The 'capacity' field is included in the `fields` tuple, but it is not a direct field of the `CinemaHall` model. Ensure that 'capacity' is defined as a property or method in the `CinemaHall` model to avoid runtime errors.